### ISSUE TITLE
SiteTree cannot locate site-tree file

### ISSUE THREAD

**Issue by mcdulltii** [mohkale/jekyll-site-tree#1]
As written in the README, even after following the specified instructions (i.e. adding `site_tree.file: map.md` to the `_config.yml`), the Jekyll build does not detect the file although `map.md` is within the source directory.

The gitlab.io link within the README is outdated due to the removal of the plugin. Hence, is it possible to clarify how the plugin reads the site_tree.file during the build process?

**Comment by mohkale** [comment]
Yeah, sorry, I stopped using jekyll altogether a while back so I kind of lost interest in this plugin. And also the permalink demonstrating how it works also doesn't work since that was linked to my jekyll site.

Looks to me like the README is out of date somewhat. Just to clarify the state of things could you share the jekyll build output & confirm that when you don't have `site_tree.file: map.md` in your config that you still have access to the `{{ page.site_tree }}` tag?

**Comment by mcdulltii** [comment]
When the config doesn't have `site_tree.file: map.md`, `bundle exec jekyll serve` still generates an empty .html on /map

**Comment by mohkale** [comment]
@mcdulltii I'll look into this when I get a chance but for now could you let me know which jekyll version you're using?

**Comment by mcdulltii** [comment]
The jekyll version i'm using is 3.6.3.
I'm currently working on my own site, so you can reference it at `mcdulltii.github.io`
